### ISSUE TITLE
Suggestion: search suggestions on "add" command

### ISSUE THREAD

**Issue by Grawl** [gamunu/vscode-yarn#4]
Would be cool to show search suggestions when I type a package name on "Yarn: Add and Save Dependency" command prompt.

**Comment by gamunu** [comment]
@Grawl yes it is. Why didn't I thought that? I'll add it to the backlog.

**Comment by gamunu** [comment]
The VSCode API doesn't support this. We'll keep this open until the issue Microsoft/vscode#21372 closes.

**Comment by Grawl** [comment]
ok good I vote there

**Comment by burtek** [comment]
> This can be done with the new API we have: https://code.visualstudio.com/updates/v1_26#_quickinput-api
